apps/backend/src/mutations/FapMutations.ts

```diff
@@ -230,7 +230,7 @@ export default class FapMutations {
     }
 
     return this.dataSource
-      .removeMemberFromFap(args, isMemberToRemoveChairOrSecretaryOfFap)
+      .removeMemberFromFap(args, args.roleId === UserRole.FAP_CHAIR)
```

Comment:
This varible is for the function in the datasource to know where to delete the member from the FAP or the Fap_reveiers table. Since sec is no longer on the FAP table we now only need to check for the Chiar role. I think ill just move this check into the datasource as it makes more sense there